yt_dlp/extractor/rozhlas.py

```diff
@@ -219,3 +244,133 @@ def _real_extract(self, url):
             'title': traverse_obj(data, ('series', 'title')),
             'entries': map(self._extract_video, data['playlist']),
         }
+
+
+class MujRozhlasIE(RozhlasBaseIE):
+    _VALID_URL = r'https?://(?:www\.)?mujrozhlas\.cz/(?:[^/]+/)*(?P<id>[^/?#&]+)'
+    _TESTS = [{
+        'url': 'https://www.mujrozhlas.cz/vykopavky/ach-jo-zase-teleci-rizek-je-mnohem-min-cesky-nez-jsme-si-mysleli',
+        'info_dict': {'id': '10739193',
+                      'title': 'Ach jo, zase to telecí! Řízek je mnohem míň český, než jsme si mysleli',
+                      'description': 'md5:db7141e9caaedc9041ec7cefb9a62908',
+                      'timestamp': 1684915200,
+                      'modified_timestamp': 1684922446,
+                      'series': 'Vykopávky',
+                      'thumbnail': 'https://portal.rozhlas.cz/sites/default/files/images/84377046610af6ddc54d910b1dd7a22b.jpg',
+                      'channel_id': 'radio-wave',
+                      'upload_date': '20230524',
+                      'modified_date': '20230524'},
+        'params': {'skip_download': True}
+    }, {
+        'url': 'https://www.mujrozhlas.cz/cetba-na-pokracovani/zena-v-polarni-noci-z-fascinujiciho-deniku-vyjimecne-odvazne-christiane',
+        'info_dict': {'id': '260da22a-1cde-3718-a1d9-4515f864c0ab',
+                      'title': 'Žena v polární noci. Z fascinujícího deníku výjimečné a odvážné Christiane Ritterové',
+                      'description': 'md5:f462de4b948bc91f7192af10bde3c279'},
+        'params': {'skip_download': True}
+    }, {
+        'url': 'https://www.mujrozhlas.cz/nespavci',
+        'info_dict': {'id': '09db9b37-d0f4-368c-986a-d3439f741f08', 'title': 'Nespavci',
+                      'description': 'md5:c430adcbf9e2b9eac88b745881e814dc'},
+        'params': {'skip_download': True}}
+    ]
+
+    _API_ROOT = 'https://api.mujrozhlas.cz'
```

Comment:
A `_call_api` method would help simplify
```suggestion
    def _call_api(self, path, item_id, msg='API JSON'):
        return self._download_json(
            f'https://api.mujrozhlas.cz/{path}/{item_id}', item_id,
            note=f'Downloading {msg}', errnote=f'Failed to download {msg}')['data']
```